package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "country-list",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
we should not change the version name in this PR.

Reply:
Yeah, thank you for your reminder @gusdecool ;)